examples/mobile/HomeApp/js/clipping-storage.js

```diff
@@ -0,0 +1,25 @@
+class Storage {
+	constructor() {
+		this.webclips = JSON.parse(localStorage.getItem("webclips")) || [];
+		this.activeWebClips = JSON.parse(localStorage.getItem("activeWebClips")) || [];
+	}
+
+	refreshStorage(element, data) {
+		this[element] = data;
+		localStorage.setItem(element, JSON.stringify(this[element]));
+	}
+
+	writeToStorage(element, link) {
+		this[element].push(link);
```

Comment:
you should replace table not expand it

Reply:
I expand it only locally but in the next line rewrite to local storage.

Reply:
each time onPopupSubmit() will be called elements will be appended to table this[element]
This will make this[element] containing the same elements many times.